src/app/core/admin/admin-entity/admin-entity.component.ts

```diff
@@ -163,6 +164,15 @@ export class AdminEntityComponent implements OnInit {
       }
       entitySchemaConfig.attributes[fieldId] = field;
     }
+    if (this.configEntitySettings) {
+      entitySchemaConfig.label = this.configEntitySettings.label;
+      entitySchemaConfig.labelPlural = this.configEntitySettings.labelPlural;
+      entitySchemaConfig.icon = this.configEntitySettings.icon;
+      const attributesString = this.configEntitySettings.toStringAttributes[0];
+      entitySchemaConfig.toStringAttributes = attributesString
+        .split(",")
+        .map((item) => item.trim());
+    }
```

Comment:
we should move this into general-settings component and expect the output to aleready have fully parsed, correct array